cdk/visit/lambda_code/log_visit/log_visit.py

```diff
@@ -38,10 +55,12 @@ def __init__(self, table, ses_client):
             self.client = ses_client
 
     def checkRegistration(self, current_user):
-        response = self.visits.query(
+        print("Checking registration for: " + current_user)
+        original_table_response = self.original.query(
             KeyConditionExpression=Key('PK').eq(current_user)
         )
-        return response['Count']
+
+        return original_table_response['Count']
```

Comment:
non blocking but why are we returning the count here?

Reply:
This is used to check and see if the user has registered before. In the original table, they would be listed as the `PK` if they have registered.